cuegui/cuegui/plugins/LogViewPlugin.py

```diff
@@ -455,7 +455,10 @@ def __init__(self, parent=None):
         # Signals are defined in code, so pylint thinks they don't exist.
         self.app.display_log_file_content.connect(self._set_log_files)
         self._log_scrollbar = self._content_box.verticalScrollBar()
-        self._log_scrollbar.valueChanged.connect(self._set_scrollbar_value)
+        # Disabled connecting the _set_scrollbar_value since it's causing the
+        # search buttons to only search within visible text.
+        # Also it doesn't really seem to do anything.
+        # self._log_scrollbar.valueChanged.connect(self._set_scrollbar_value)
```

Comment:
I think this can be removed altogether and the comment can be moved to the commit message.

Reply:
Please let e know if you intend to remove this before I merge the changes in.

Reply:
I'll remove it then